collectors/0/kafka_consumer_jmx.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+import re
+import sys
+
+from collectors.lib.jmx_monitor import JmxMonitor
+
+MONITORED_MBEANS = ["kafka.consumer", ""]
+
+
+class KafkaConsumerJmxMonitor(JmxMonitor):
+    USER = "flume"
+    PROCESS_NAME = "org.apache.flume.node.Application"
+    METRIC_PREFIX = "flume"
```

Comment:
what i meant is group all kafka jmx processing in one util class. then make use of this class in actual collectors. the specifics like process name, kafka roles, etc. should be left in individual collectors. i would imagine we end up with these collectors:
- kafka_broker
- flume or flume_jmx
- flume_kafka (i know you want to separate this into two, but i feel it's too specific because what role this flaka host plays is really up to our configuration. if we decide to have both consumer and producer on a host at some point, i don't want us having to create a third role)

on a side note, i assume you have checked it's ok to make multiple connections to the same jmx server?